tests/resources/mocks/data_provider_mock.py

```diff
@@ -72,55 +75,60 @@ def build_download_endpoint(provider_uri=None):
         return "GET", f"{provider_uri}/api/v1/services/{service_name}"
 
     @staticmethod
-    def download_service(
-        did,
-        service_endpoint,
-        wallet,
-        files,
-        destination_folder,
-        service_id,
-        token_address,
-        order_tx_id,
-        index=None,
-        userdata=None,
+    def initialize(
+        did: str,
+        service_id: str,
+        file_index: int,
+        consumer_address: str,
+        service_endpoint: str,
+        userdata: Optional[Dict] = None,
     ):
+        initialize_endpoint = service_endpoint
+        initialize_endpoint += f"?documentId={did}"
+        initialize_endpoint += f"&serviceId={service_id}"
+        initialize_endpoint += f"&fileIndex={file_index}"
+        initialize_endpoint += f"&consumerAddress={consumer_address}"
+        if userdata:
+            initialize_endpoint += f"&userdata={json.dumps(userdata)}"
+        DataServiceProvider._http_method("get", initialize_endpoint)
```

Comment:
I don't understand the point. Why use a mock when you eventually use the main DataServiceProvider? This mock is just a passthrough.

Reply:
I will delete those methods from the mock class.